packages/docs/docs/guides/8-glossary.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Glossary
+keywords:
+---
+
+# Ecosystem
+
+## Request Portal API
+
+The Request Portal is a software built on top of the Request Protocol that allows third party software to easily integrate Request, all the private keys to create requests and perform actions to them are stored by Request and can be managed through OAuth. The Request Portal API is an API built on top of the Request Portal.
+
+## Request Client
+
+The Request Client is a Javascript library made to interact directly with the Request Protocol. The Request Client connects to a Request Node.
+
+## Request Node
+
+Request Nodes are HTTP servers exposing an API used to allow Request Client to communicate with the Request Protocol. These servers abstract the complexity of IPFS and Ethereum used by the current implementation of the Request Protocol.
```

Comment:
I don't think "the current implementation" makes sense here. Yes, we could change one day, but it's not planned. It could confuse users